gnoi_os/gnoi_os.go

```diff
@@ -0,0 +1,95 @@
+/* Copyright 2020 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"flag"
+	"time"
+
+	log "github.com/golang/glog"
+	gnoiOS "github.com/google/gnxi/gnoi/os"
+	"github.com/google/gnxi/utils/credentials"
+	"google.golang.org/grpc"
+)
+
+var (
+	targetAddr = flag.String("target_addr", ":9339", "The target address in the format of host:port")
+	targetName = flag.String("target_name", "", "The target name used to verify the hostname returned by TLS handshake")
+	version    = flag.String("version", "", "Version of the OS required when using the activate operation")
+	osFile     = flag.String("os", "", "Path to the OS image for the install operation")
+	op         = flag.String("op", "", "OS service operation. Can be one of: install, activate, verify")
+	timeOut    = flag.Duration("time_out", 5*time.Second, "Timeout for the operation, 5 seconds by default")
```

Comment:
just an FYI that we may need to test that this timeout works for installing large OS images.

Reply:
Thats true